build/constants.yaml

```diff
@@ -26,7 +26,7 @@
     dot-net-core-31-sdk-version: 3.1.412
     dot-net-core-50-sdk-version: 5.0.302
     dot-net-50-sdk-version: 5.0.400
-    dot-net-60-sdk-version: 6.0.100-preview.7.21379.14
+    dot-net-60-sdk-version: 6.0.100-rc.1
```

Comment:
the full version is 6.0.100-rc.1.21458.32

Reply:
Updated, thank you